UI/users/preferences.html

```diff
@@ -69,7 +70,8 @@
                                   options = user.dateformats
                                   default_values = [ user.prefs.dateformat ]
                                   text_attr = 'format'
-                                  value_attr = 'format'
+                                  value_attr = 'format',
```

Comment:
None of the other lines ends in a comma. Why should this one?

Reply:
Fixed